greeclimate/device.py

```diff
@@ -155,6 +161,13 @@ class Device(DeviceProtocol2, Taskable):
         water_full: A bool to indicate the water tank is full
     """
 
+    """ Device properties """
+    hid = None
+    version = None
+    check_version = True
+    _properties = {}
+    _dirty = []
+
     def __init__(self, device_info: DeviceInfo, timeout: int = 120, loop: AbstractEventLoop = None):
```

Comment:
Setting timeout to 120 means the binding attempt has to wait 120 seconds before it times out and tries the new cipher.
I am not if we should lower the timeout here or if it is possible to set a different timeout for the binding itself.

Reply:
This is the socket timeout, it probably should stay at this value, but I can add another timeout to the bind response specifically as an exception.